### ISSUE TITLE
Variable was not processed correctly

### ISSUE THREAD

**Issue by leonardfischer** [scssphp/scssphp#169]
Hey,

first of all: thank you guys for this awesome library! You are doing a great job :)

But then - with the latest release I found an issue with the compiler. This is my input (from https://bootswatch.com/cosmo/):

```scss
$web-font-path: "https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;700&display=swap" !default;
@import url($web-font-path);

[...]
```

And this is the output - I use the `Compressed` formatter:

```css
@import url($web-font-path);[...]
```

With the previous version it worked fine.

Best regards,
Leo

**Comment by Cerdic** [comment]
Indeed, the faulty commit is 84b8a0099adea862db2cbcbda95d4ee148827abc

**Comment by Cerdic** [comment]
This is fixed by f6f9167, sorry for the mistake. 

There has been some big refactoring in the last release, and looks like some untested border cases have been broken (despite the big enlargement of the test base thanks to @stof )

**Comment by stof** [comment]
@Cerdic should we submit tests in sass-spec for such case ?
